go.mod

```diff
@@ -70,6 +70,7 @@ require (
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/jonboulle/clockwork v0.1.0 // indirect
 	github.com/json-iterator/go v1.1.6 // indirect
+	github.com/jteeuwen/go-bindata v0.0.0-20180305030458-6025e8de665b // indirect
```

Comment:
why is this reappearing? i just removed it due to the release build crash. let's stick with vfsgen.

Reply:
it was added when I built locally, anyway I removed it.